bigquery_storage/to_dataframe/main_test.py

```diff
@@ -0,0 +1,178 @@
+
+    # Download a table.
+    table = bigquery.TableReference.from_string(
+        "bigquery-public-data.utility_us.country_code_iso"
+    )
+    rows = bqclient.list_rows(
+        table,
+        selected_fields=[
+            bigquery.SchemaField("country_name", "STRING"),
+            bigquery.SchemaField("fips_code", "STRING"),
+        ],
+    )
+    dataframe = rows.to_dataframe(bqstorageclient)
+    print(dataframe.head())
+    # [END bigquerystorage_pandas_tutorial_read_table]
+
+    out, _ = capsys.readouterr()
+    assert "country_name" in out
+
+
+def test_query_to_dataframe(capsys, clients, temporary_dataset):
+    bqclient, bqstorageclient, _ = clients
+    dataset_id = temporary_dataset
+
+    # [START bigquerystorage_pandas_tutorial_read_query_results]
+    import uuid
+
+    from google.cloud import bigquery
+
+    # Due to a known issue in the BigQuery Storage API (TODO: link to
+    # public issue), small query result sets cannot be downloaded. To
+    # workaround this issue, write results to a destination table.
+
+    # TODO: Set dataset_id to a dataset that will store temporary query
+    #       results. Set the default table expiration time to ensure data is
+    #       deleted after the results have been downloaded.
+    # dataset_id = "temporary_dataset_for_query_results"
+    dataset = bqclient.dataset(dataset_id)
+    table_id = "queryresults_" + uuid.uuid4().hex
+    table = dataset.table(table_id)
+
+    # Download query results.
+    query_string = """
+    SELECT
+    CONCAT(
+        'https://stackoverflow.com/questions/',
+        CAST(id as STRING)) as url,
+    view_count
+    FROM `bigquery-public-data.stackoverflow.posts_questions`
+    WHERE tags like '%google-bigquery%'
+    ORDER BY view_count DESC
+    """
+    query_config = bigquery.QueryJobConfig(
+        destination=table, write_disposition="WRITE_TRUNCATE"
+    )
+
+    dataframe = (
+        bqclient.query(query_string, job_config=query_config)
+        .result()
+        .to_dataframe(bqstorageclient)
+    )
+    print(dataframe.head())
+    # [END bigquerystorage_pandas_tutorial_read_query_results]
+
+    out, _ = capsys.readouterr()
+    assert "stackoverflow" in out
+
+
+def test_session_to_dataframe(capsys, clients):
+    bqclient, bqstorageclient, project_id = clients
+
+    # [START bigquerystorage_pandas_tutorial_read_session]
+    from google.cloud import bigquery_storage_v1beta1
+
+    table = bigquery_storage_v1beta1.types.TableReference()
+    table.project_id = "bigquery-public-data"
+    table.dataset_id = "new_york_trees"
+    table.table_id = "tree_species"
+
+    # Select columns to read with read options. If no read options are
+    # specified, the whole table is read.
+    read_options = bigquery_storage_v1beta1.types.TableReadOptions()
+    read_options.selected_fields.append("species_common_name")
+    read_options.selected_fields.append("fall_color")
+
+    parent = "projects/{}".format(project_id)
```

Comment:
This might be confusing to the user if they don't know it's their project ID, not the "bigquery-public-data" project above. What about using `bqstorageclient.project` or `bqclient.project` instead?

Reply:
Renamed the variable to `your_project_id`